homeassistant/components/saj/sensor.py

```diff
@@ -162,14 +167,18 @@ def remove_listener():
 class SAJsensor(Entity):
     """Representation of a SAJ sensor."""
 
-    def __init__(self, pysaj_sensor):
+    def __init__(self, pysaj_sensor, inverter_name=""):
         """Initialize the sensor."""
         self._sensor = pysaj_sensor
+        self._inverter_name = inverter_name
         self._state = self._sensor.value
 
     @property
     def name(self):
         """Return the name of the sensor."""
+        if self._inverter_name != "":
+            return f"saj_{self._inverter_name}_{self._sensor.name}"
```

Comment:
If there's no inverter name, won't this create a strange string with a double underscore in the middle? Or is that okay?

Reply:
No actually I'm checking if the string is not empty, so if it is specified then add it to the name.
If empty then it well use the return statement 2 lines below.

I've changed this to make use of None instead of empty string.